app/classifier/tasks/metadata-based-feedback.cjsx

```diff
@@ -0,0 +1,69 @@
+React = require 'react'
+
+module.exports = React.createClass
+  displayName: 'MetadataBasedFeedback'
+
+  withinTolerance: (userX, userY, metaX, metaY, tolerance) ->
+    distance = Math.sqrt((userY - metaY)**2 + (userX - metaX)**2)
+    isWithinTolerance = distance < tolerance
+    return isWithinTolerance
```

Comment:
This is just an fyi about coffeescript: returns are implicit, so this line can just be `isWithinTolerance`